docs/synthetics/using-the-api.md

```diff
@@ -1,12 +1,17 @@
-title: Using the API.
-description: How to use the Synthetics API.
+title: Using the Synthetics API for triggering and creating monitors
+description: How to use the Synthetics API for triggering and creating monitors.
```

Comment:
"Using the Synthetics API for fetching and creating monitors" sounds a bit better to me